### DISCUSSION TITLE
Donations to zim-wiki

### DISCUSSION THREAD

**Discussion by treborxy** [zim-desktop-wiki/zim-desktop-wiki#1710] [Ideas]
That zim is a most useful tool is a commonplace - and that donations are welcome should be too.

So - how to spread the word? A hint on the [website](https://www.zim-wiki.org/contribute.html) is a nice step - but what could be more?

Any suggestions welcome! 

How about [librapay](https://liberapay.com/)? 

Gratefully
Robert

**Comment by jaap-karssenberg** [comment]
Thanks, but no thanks. As I explain in the website and in more detail in
the wiki WhyIsItNotDoneYet · zim-desktop-wiki/zim-desktop-wiki Wiki · GitHub
<https://github.com/zim-desktop-wiki/zim-desktop-wiki/wiki/WhyIsItNotDoneYet> ,
donations do not increase the amount of work done on zim directly. So I
don't want to solicit for more donations at this time.

Kind regards,

Jaap

**Reply by treborxy** [reply]
Ok - thank you for zim anyway. :-)

**Comment by rzelnik** [comment]
I am a happy Zim user for many years and I would like to keep the project alive forever (or, at least, for many years in the future). I understand that it is difficult to process all the issues and all the feature requests if you are (mostly) a sole developer. Jaap, would you like to expand your team? I don't offer myself as a developer because I don't know Python. What about allowing donations again (currently the PayPal button is broken) and invite more developers to contribute for bounties? There are many great tools for managing donations. Github itself allows to collect donations, and there is a great [Open Collective](https://opencollective.com/) service that allows you to create a virtual non-profit organization around your open source project with transparent payment gateway and many other cool features.

> Monetary donations: Since I'm fulltime employed otherwise, donations do not help me to free up more time. In the past I have tried using donations to put "bounties" on wanted features, but this failed to attract any substantial submissions.

How did you try to attract the developers? Were the bounties high enough to attract them? Could you please publish the list of wanted features with bounties again? I would like to help you with that.

**Reply by introt** [reply]
Not sure the numbers work out in favor of bounties. Most of the feature requests require at least tens of hours of work - either due to the complexity of the implementation, or the required prior knowledge about the existing code base - while the bounties generally top out at a few hours pay. Even if we could pool together enough money to attract appropriate talent, who's going to bug fix and maintain all those one-off features moving forward?

Having contributors work on stuff they themselves use is imo much more sustainable at this scale. Reviewing and reworking mercenary code would just introduce more strain on the project, and likely fasten its demise.
